mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -146,6 +146,7 @@ impl UtxoRpcClientEnum {
         check_every: u64,
     ) -> Box<dyn Future<Item = (), Error = String> + Send> {
         let selfi = self.clone();
+        let mut retry_req_count = 10;
```

Comment:
Can we make a `const` equal to 10 and use it here? it can be named `TX_NOT_FOUND_RETRIES` or something in this regard.